model/client.go

```diff
@@ -831,8 +831,10 @@ func (c *Client) Command(channelId string, command string) (*Result, *AppError)
 		return nil, err
 	} else {
 		defer closeBody(r)
+
+		response, _ := CommandResponseFromJson(r.Body)
```

Comment:
I'm ignoring the error here, because I'm assuming our own server can generate correct JSON. The additional error handling is primarily meant to help debug user-driven JSON.